crates/ironrdp-pdu/src/rdp/suppress_output.rs

```diff
@@ -0,0 +1,57 @@
+use std::io;
+
+use byteorder::{ReadBytesExt as _, WriteBytesExt as _};
+use num_derive::{FromPrimitive, ToPrimitive};
+use num_traits::{FromPrimitive, ToPrimitive};
+
+use crate::{geometry::InclusiveRectangle, PduParsing};
+
+#[repr(u8)]
+#[derive(Debug, PartialEq, Eq, FromPrimitive, ToPrimitive, Clone)]
+pub enum AllowDisplayUpdatesType {
+    SuppressDisplayUpdates = 0x00,
+    AllowDisplayUpdates = 0x01,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct SuppressOutputPdu {
```

Comment:
Note: FYI, we are now trying to make cross-referencing with spec easier when implementing new PDUs
https://github.com/Devolutions/IronRDP/blob/d69ff6e6e7ebf76864f4d812a33677b8d0d70eac/STYLE.md#doc-comments-should-link-to-reference-documents